sources/core/objects/Floor.cpp

```diff
@@ -15,10 +15,10 @@ Floor::Floor(quint32 id) : ITurf(id)
     SetPassable(D_ALL, Passable::FULL);
 
     SetSprite("icons/floors.dmi");
-    SetState("floor");
```

Comment:
You should keep that line, because otherwise the floor object will be invisible in the map editor.